src/main/java/net/neoforged/neoforge/registries/DeferredRegister.java

```diff
@@ -164,7 +178,9 @@ public static DeferredRegister.Blocks createBlocks(String modid) {
      * @see #create(ResourceKey, String)
      * @see #create(ResourceLocation, String)
      * @see #createItems(String)
+     * @deprecated Use the overload with the {@link ResourceKey} parameter
      */
+    @Deprecated(since = "21.1", forRemoval = true)
```

Comment:
I believe we still use the full Minecraft version in the deprecation annotations.
```suggestion
    @Deprecated(since = "1.21.1", forRemoval = true)
```

(Same comment for the `DataComponents` constructor.)

Reply:
From searching the code base, it was very inconsistent. Both version schemes were used throughout neos code and patches. I personally preferred using neo's form

Reply:
It doesn't seem to be very inconsistent to me, based on my search. Out of the roughly 30 or so `@Deprecated` annotations in NeoForge with `since` values (both its own code and in patches), there are only five annotations whose `since` value is in the same format as NeoForge's versioning: two in `IEntityExtension`, one in `ScreenEvent`, one in `IItemExtension`, and one in `IItemStackExtension`.

That seems to suggest that while there is some inconsistency, the vast majority of annotations still refer to the Minecraft version format rather than the NeoForge version format. However, for future consistency, I'll bring it up in the Discord with the team to see what's preferred for such annotations going forward.

Reply:
I'd argue that's enough for the label of "very inconsistent", though I will adjust it to the more common format.

Reply:
Per discussion [in Discord with the team](https://discord.com/channels/313125603924639766/1154167065519861831/1282456685008523399), the format for the `since` value in the `@Deprecated` annotation should be the Minecraft version format, so `since = "1.21.1"`.